### DISCUSSION TITLE
How to use Coquii STT with GPU on Colab?

### DISCUSSION THREAD

**Discussion by jeff-bolton** [coqui-ai/STT#2078] [General]
How do you get coqui to run on the GPU in Colab?

The Coqui example notebooks run fine, but none of them use the GPU.

Is there some configuration setting that needs to be made (and documented) to get this to work?

Thanks in advance!

**Comment by JRMeyer** [comment]
hi @jeff-bolton 👋 

Thanks for posting this question!

If you want to use the GPU for training, you should uninstall tensorflow and reinstall tensorflow-gpu, as such:

```
! pip uninstall tensorflow
! pip install "tensorflow-gpu==1.15"
```

**Comment by hammondm** [comment]
Hi
I'm having the same issue. I tried the move above and still not working.
mike h.
